### PR TITLE
[Breaking change] Migrate from AAD Graph to Microsoft Graph

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description
- Removed warning messages
- Removed duplicated cmdlets
- [Breaking change] Migrated from AAD Graph to Microsoft Graph
- [Breaking change] Changed the return id in PSDenyAssignment from Guid string to fully qualified id
- Allowed `Id` in `Get-AzDenyAssignment` to accept fully qualified id

Migration guide:
`Id` in `Get-AzDenyAssignment` still accept guid, no change for input.
The output of `Get-AzDenyAssignment` changes: Id changes from "22704996-fbd0-4ab1-8625-722d897825d2" -> "/subscriptions/4004a9fd-d58e-48dc-aeb2-4a4aec58606f/providers/Microsoft.Authorization/denyAssignments/22704996-fbd0-4ab1-8625-722d897825d2"
```powershell
Get-AzDenyAssignment
```
```output
Id                      : /subscriptions/4004a9fd-d58e-48dc-aeb2-4a4aec58606f/providers/Microsoft.Authorization/denyAssignments/22704996-fbd0-4ab1-8625-722d897825d2
DenyAssignmentName      : Test deny assignment 1
Description             : Test deny assignment for PS cmdlets
Actions                 : {foo/*}
NotActions              : {foo/*/read}
DataActions             : {foo/*}
NotDataActions          : {}
Scope                   : /subscriptions/4004a9fd-d58e-48dc-aeb2-4a4aec58606f
DoNotApplyToChildScopes : False
Principals              : {
                          DisplayName:  All Principals
                          ObjectType:   SystemDefined
                          ObjectId:     00000000-0000-0000-0000-000000000000
                          }
ExcludePrincipals       : {
                          DisplayName:  testuser
                          ObjectType:   User
                          ObjectId:     f8d526a0-54eb-4941-ae69-ebf4a334d0f0
                          }
IsSystemProtected       : True
```

Benefit for this change: scope can be included in Id

To get guid as before: `Id.TrimEnd('/').Substring(Id.LastIndexOf('/') + 1)`

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Resources/Resources/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,8 @@
 -->
 
 ## Upcoming Release
+* [Breaking change] Migrate from AAD Graph to Microsoft Graph
```
Comment (BethanyZhou) [inline src/Resources/Resources/ChangeLog.md:22]:
```suggestion
* [Breaking change] Migrated from AAD Graph to Microsoft Graph 
```

File: src/Resources/Resources/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,10 @@
 -->
 
 ## Upcoming Release
+* [Breaking change] Migrated from AAD Graph to Microsoft Graph
+* [Breaking change] Changed the return id in PSDenyAssignment from Guid string to fully qualified id
+* Allowed `Id` in `Get-AzDenyAssignment` to accept fully qualified id
+* Removed breaking change warnings for AAD cmdlets
```
Comment (isra-fel) [inline src/Resources/Resources/ChangeLog.md:25]:
```suggestion
```
This is reorgnization of internal code, not so helpful to our customers

File: src/Resources/Resources/Models.Authorization/AuthorizationClientExtensions.cs [hunk 3]
```diff
@@ -314,7 +313,8 @@ private static IEnumerable<PSRoleAssignment> ToPSRoleAssignments(this IEnumerabl
                         RoleDefinitionId = roleDefinition.Id,
                         RoleDefinitionName = roleDefinition.Name,
                         Scope = assignment.Scope,
-                        ObjectType = assignment.Type,
+                        // No idea about the object type in exception scenario
+                        // ObjectType = assignment.Type
```
Comment (isra-fel) [inline src/Resources/Resources/Models.Authorization/AuthorizationClientExtensions.cs:317]: ...

File: src/Resources/Resources/Properties/Resources.Designer.cs [hunk 4]
```diff
@@ -268,7 +268,7 @@ internal class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to The User/Application doesn&apos;t have sufficient AAD graph permissions to perform the operation..
+        ///   Looks up a localized string similar to The User/Application doesn&apos;t have sufficient Microsoft AD graph permissions to perform the operation..
```
Comment (isra-fel) [inline src/Resources/Resources/Properties/Resources.Designer.cs:271]:
```suggestion
        ///   Looks up a localized string similar to The User/Application doesn&apos;t have sufficient Microsoft Graph permissions to perform the operation..
```

File: src/Resources/Resources/help/Get-AzDenyAssignment.md [hunk 5]
```diff
@@ -383,7 +383,7 @@ Accept wildcard characters: False
 ```
 
 ### -Id
-Deny assignment id.
+Deny assignment fully qualified id or guid. When Id is provided as a guid, will take current subscription as default scope.
```
Comment (isra-fel) [inline src/Resources/Resources/help/Get-AzDenyAssignment.md:386]:
```suggestion
Deny assignment fully qualified ID or GUID. When Id is provided as a GUID, will take current subscription as default scope.
```

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: src/Resources/Resources/Models.Authorization/AuthorizationClientExtensions.cs (hunk 3 above)
Comment (BethanyZhou) [inline src/Resources/Resources/Models.Authorization/AuthorizationClientExtensions.cs:317]:
```suggestion
```

**Review by isra-fel — APPROVED** [review]

**Pull request merged**